### ISSUE TITLE
Add row for average across a selected group

### ISSUE THREAD

**Issue by za158** [georgetown-cset/parat#101]
If a group is included in the active view, a row should appear that displays the average for the whole group, so that (eg) it’s easy to see by sorting if a given company is above or below the average.

**Comment by brianlove** [comment]
The underlying data needed for this and #84 seem to be the same (at least from my current understanding)

**Comment by jmelot** [comment]
@brianlove how would you like this data to be provided? Intuitively it makes sense for me to treat these groups like just another "company", but that might make their data annoying to access.

**Comment by brianlove** [comment]
Let's adjust the `overall.groups` object and put the relevant data there:

```js
overall = {
  groups: {
    sp500: {
      name: "S&P 500",
      articles: { /* Article data */ },
      patents: { /* Patent data */ },
      // etc...
    },
    global500: { /* etc */ },
  },
};
```

There's one minor change that would be needed to the GUI to adjust for the fact that the `groups` object won't directly point to the name, but that won't be a major issue.

**Comment by brianlove** [comment]
Closed by #183
